kitt/image/objdetect/render.py

```diff
@@ -3,40 +3,38 @@
 import cv2
 import numpy as np
 
-from .annotation import AnnotatedBBox, AnnotatedImage
-
-
-def render_annotated_image(
-    annotated: AnnotatedImage, color=(0, 0, 255), line_width=3
-) -> np.ndarray:
-    """Render an image along with its annotations."""
-    assert annotated.image
-    image = annotated.to_numpy()
-    width, height = annotated.width, annotated.height
-
-    for annotation in annotated.annotations:
-        bbox = annotation.bbox.denormalize(width, height).to_int()
-        cv2.rectangle(image, bbox.top_left, bbox.bottom_right, color, line_width)
-    return image
+from .annotation import AnnotatedBBox, AnnotationType
+from ..image import get_image_size
 
 
 def render_annotations(image: np.ndarray, annotations: List[AnnotatedBBox]):
```

Comment:
It would be nice to have a function like `color = get_bbox_color(bbox)` in the parameters, but it's a detail.